src/test/java/bio/terra/service/job/StairwayExceptionSerializerTest.java

```diff
@@ -0,0 +1,32 @@
+package bio.terra.service.job;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.*;
+
+import bio.terra.common.category.Unit;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.context.ActiveProfiles;
+
+@ActiveProfiles({"google", "unittest"})
+@Tag(Unit.TAG)
+public class StairwayExceptionSerializerTest {
```

Comment:
It's a minor thing but in junit 5 you don't need to make the test class or method `public`. Sonar will flag this as a warning, if you want to see sonar errors in the IDE you can install sonarlint which will flag this line.
```suggestion
class StairwayExceptionSerializerTest {
```

Reply:
Oh, right! Thanks! 
I hadn't heard of the sonarlint plugin - definitely helpful to see these sort of warnings before it's flagged in the sonar report in the PR. 
![Screenshot 2024-01-31 at 9 37 02 PM](https://github.com/DataBiosphere/jade-data-repo/assets/13254229/23a35031-bb94-4d68-b1fe-065758e55eb9)